dash_player/DashPlayer.py

```diff
@@ -95,6 +95,9 @@ def __init__(self, id=Component.UNDEFINED, className=Component.UNDEFINED, url=Co
         _explicit_args = kwargs.pop('_explicit_args')
         _locals = locals()
         _locals.update(kwargs)  # For wildcard attrs and excess named props
-        args = {k: _locals[k] for k in _explicit_args}
-
+        args = {k: _locals[k] for k in _explicit_args if k != 'children'}
+        for k in []:
+            if k not in args:
+                raise TypeError(
+                    'Required argument `' + k + '` was not specified.')
```

Comment:
This looks like it was built with an older version of dash? Newer Dash strips this block out when it would be a noop like this (`for k in []`)

Reply:
Ah, yeah, looks like I was using 2.6.1. Rebuilt it now. Thanks!